src/Reflection/Pattern.agda

```diff
@@ -25,9 +25,41 @@ open import Reflection.Argument
 ------------------------------------------------------------------------
 -- Re-exporting the builtin type and constructors
 
-open import Agda.Builtin.Reflection public using (Pattern)
+open import Agda.Builtin.Reflection as Builtin public using (Pattern)
 open Pattern public
 
+------------------------------------------------------------------------
+-- Showing
+
+mutual
+
+  showPatterns : List (Arg Pattern) → String
+  showPatterns [] = ""
+  showPatterns (a ∷ ps) = showArg a ++ showPatterns ps
+    where
+      showRel : Builtin.Relevance → String
+      showRel Builtin.relevant = ""
+      showRel Builtin.irrelevant = "."
+
+      showArg : Arg Pattern → String
```

Comment:
These should probably go in the appropriate `Reflection.Argument(.Relevance/Visibility)`.

Reply:
There are more verbose show functions in `.Relevance` and `.Visibility`, here I wanted something more lightweight. I put them in a where-clause since they are not very useful on their own.

Reply:
Ah, I see.